components/dot_menu/dot_menu.jsx

```diff
@@ -255,6 +257,28 @@ export default class DotMenu extends React.PureComponent {
         );
     }
 
+    onClickLocation = async (item) => {
+        const response = await doPluginCall({
+            form_url: item.form_url,
+            context: {
+                app_id: item.app_id,
+                team_id: this.props.teamId,
+                channel_id: this.props.post.channel_id,
+                post_id: this.props.post.id,
+            },
+            from: [
+                {
+                    item,
+                },
+            ],
+        });
+        if (response.type === 'command') {
+            response.data.args.command = response.data.command;
+            this.props.actions.executeCommand(response.data.command, response.data.args);
```

Comment:
Do you think this should be handled in `doPluginCall` instead?

Reply:
The problem is that, in order to executeCommand to work, has to be dispatched to the redux store, and I did not know a better way to do it. :sweat_smile:

Reply:
`doPluginCall` can be passed in as a dispatched function through the `index.js` file

Reply:
:facepalm: I will update this along with the refactoring...